internal/request/graphql/schema/descriptions.go

```diff
@@ -33,8 +33,8 @@ var (
 		client.FieldKind_NILLABLE_STRING:       gql.String,
 		client.FieldKind_STRING_ARRAY:          gql.NewList(gql.NewNonNull(gql.String)),
 		client.FieldKind_NILLABLE_STRING_ARRAY: gql.NewList(gql.String),
-		client.FieldKind_NILLABLE_BLOB:         schemaTypes.BlobScalarType(),
```

Comment:
question: Why have you made this change?  This was done quite recently as these are mutated by the gql library and making them singletons results in shared mutated state between database instances.

Reply:
this is done to follow the same style that is used by `github.com/sourcenetwork/graphql-go`: to have 1 scalar definition instance instead of creating always a new.

Reply:
This is wrong (and `graphql-go`), and we should not do it.  As well as being a production bug, it prevents us from using `t.Parallel()` in our tests.

Reply:
If we have issues with `t.Paralel()` we should change how we use `gql.String` and others. I just made it consistent.

I tried to roll this part back and see if it works. It doesn't because there is a pointer comparison https://github.com/sourcenetwork/graphql-go/blob/master/schema.go#L295

Seems like the change is required for the fix.

Reply:
It is not just about `t.Parallel()`.   `graphql-go` is very singleton heavy, and it mutates the singleton types we provide it.  This means multiple embedded Defra instances leak GQL types between them, instance1 will have bits of instance2's gql types in its GQL schema.

> It doesn't because there is a pointer comparison 

You need to only call `schemaTypes.BlobScalarType()` once per database instance, and fetch it from the type map where-ever else you require it.  All our other types do this.